backend/dataall/api/Objects/Dashboard/resolvers.py

```diff
@@ -1,7 +1,10 @@
+import os
 from .... import db
 from ....api.constants import DashboardRole
 from ....api.context import Context
 from ....aws.handlers.quicksight import Quicksight
+from ....aws.handlers.parameter_store import ParameterStoreManager
+from ....aws.handlers.sts import SessionHelper
```

Comment:
Is this import module used?

Reply:
ParameterStoreManager yes, SessionHelper no. I am removing it